Modules/Quotas/src/Quotas.Infrastructure/Persistence/Database/EntityConfigurations/TierQuotaEntityTypeConfiguration.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Backbone.Modules.Quotas.Domain.Aggregates.Identities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Backbone.Modules.Quotas.Infrastructure.Persistence.Database.EntityConfigurations;
+
+public class TierQuotaEntityTypeConfiguration : IEntityTypeConfiguration<TierQuota>
+{
+    public void Configure(EntityTypeBuilder<TierQuota> builder)
+    {
+        builder.UseTpcMappingStrategy();
```

Comment:
Is this line necessary?

Reply:
Just tested, it is not.
I'll remove it.